auth/session.go

```diff
@@ -82,11 +91,19 @@ func (auth *Authenticator) CreateSession(username string, ttl time.Duration) (*L
 		return nil, err
 	}
 
+	user, err := auth.GetUser(username)
+	if user != nil && user.Disabled() {
+		return nil, base.HTTPErrorf(400, "User is disabled")
```

Comment:
Is the disabled handling required here?  I don't think we have a reason to remove the ability to create sessions for disabled users?

Reply:
I think I copy/pasted from a different function, they'd fail in authentication anyway so this is fine.